gce-staging-registry-1/main.tf

```diff
@@ -0,0 +1,56 @@
+variable "env" {
+  default = "staging"
+}
+
+variable "gce_gcloud_zone" {}
+
+variable "index" {
+  default = 1
+}
+
+variable "project" {
+  default = "eco-emissary-99515"
+}
+
+variable "syslog_address_com" {}
+variable "syslog_address_org" {}
+
+variable "travisci_net_external_zone_id" {
+  default = "Z2RI61YP4UWSIO"
```

Comment:
It's a trivial thing, but I would like to get away from referencing this directly and instead looking it up as with https://github.com/travis-ci/terraform-config/blob/9ee6c353d33d1589229c719db74f2a41761a6fe0/dns-production-0/main.tf#L45-L47